templates/tfengine/components/cicd/triggers.tf

```diff
@@ -23,6 +23,11 @@
   {{- $terraform_root_prefix = ""}}
 {{- end}}
 
+{{- $worker_pool := ""}}
+{{- if has . "worker_pool"}}
+  {{- $worker_pool := printf "projects/%s/locations/%s/workerPools/%s" .worker_pool.project .worker_pool.location .worker_pool.name}}
```

Comment:
need to move to inside of `{{range get . "envs" -}}` so per-env can be achieved

Reply:
done